### PR TITLE
plasma-infra: delete unnecessary examples workflows

### PR DESCRIPTION
**Pull request by Yakutoc**
Приняли решение удалить за ненадобностью пакет - examples.   

:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.543.5197189327.swift)  
[color_sberHealth_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.543.5197189327.kt)  
[color_sberHealth_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.543.5197189327.ts)  
[color_sbermarket_business_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.543.5197189327.swift)  
[color_sbermarket_business_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.543.5197189327.kt)  
[color_sbermarket_business_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.543.5197189327.ts)  
[color_sbermarket_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.543.5197189327.swift)  
[color_sbermarket_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.543.5197189327.kt)  
[color_sbermarket_metro_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.543.5197189327.swift)  
[color_sbermarket_metro_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.543.5197189327.kt)  
[color_sbermarket_metro_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.543.5197189327.ts)  
[color_sbermarket_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.543.5197189327.ts)  
[color_sbermarket_selgros_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.543.5197189327.swift)  
[color_sbermarket_selgros_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.543.5197189327.kt)  
[color_sbermarket_selgros_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.543.5197189327.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.543.5197189327.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.543.5197189327.kt)  
[color_sbermarket_wlbusiness_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.543.5197189327.ts)  
[color_sberonline_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.543.5197189327.swift)  
[color_sberonline_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.543.5197189327.kt)  
[color_sberonline_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.543.5197189327.ts)  
[color_sberprime_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.543.5197189327.swift)  
[color_sberprime_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.543.5197189327.kt)  
[color_sberprime_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.543.5197189327.ts)  
[shadow_sbermarket_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.543.5197189327.ts)  
[typo_mage_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.543.5197189327.swift)  
[typo_mage_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.543.5197189327.kt)  
[typo_mage_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.543.5197189327.ts)  
[typo_plasma_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.543.5197189327.swift)  
[typo_plasma_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.543.5197189327.kt)  
[typo_plasma_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.543.5197189327.ts)  
[typo_ruler_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.543.5197189327.swift)  
[typo_ruler_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.543.5197189327.kt)  
[typo_ruler_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.543.5197189327.ts)  
[typo_sage_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.543.5197189327.swift)  
[typo_sage_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.543.5197189327.kt)  
[typo_sage_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.543.5197189327.ts)  
[typo_sbermarket_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.543.5197189327.swift)  
[typo_sbermarket_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.543.5197189327.kt)  
[typo_sbermarket_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.543.5197189327.ts)  
[typo_soulmate_ios-swift--canary.543.5197189327.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.543.5197189327.swift)  
[typo_soulmate_kotlin--canary.543.5197189327.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.543.5197189327.kt)  
[typo_soulmate_react-native--canary.543.5197189327.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.543.5197189327.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.216.1-canary.543.5197189327.0
  npm install @salutejs/plasma-core@1.121.1-canary.543.5197189327.0
  npm install @salutejs/plasma-hope@1.216.1-canary.543.5197189327.0
  npm install @salutejs/plasma-icons@1.150.1-canary.543.5197189327.0
  npm install @salutejs/plasma-temple@1.165.1-canary.543.5197189327.0
  npm install @salutejs/plasma-tokens-b2b@1.21.1-canary.543.5197189327.0
  npm install @salutejs/plasma-tokens-b2c@0.30.1-canary.543.5197189327.0
  npm install @salutejs/plasma-tokens-web@1.36.1-canary.543.5197189327.0
  npm install @salutejs/plasma-tokens@1.55.1-canary.543.5197189327.0
  npm install @salutejs/plasma-ui@1.197.1-canary.543.5197189327.0
  npm install @salutejs/plasma-web@1.216.1-canary.543.5197189327.0
  npm install @salutejs/plasma-cy-utils@0.62.1-canary.543.5197189327.0
  npm install @salutejs/plasma-sb-utils@0.119.1-canary.543.5197189327.0
  npm install @salutejs/plasma-tokens-utils@0.26.1-canary.543.5197189327.0
  # or 
  yarn add @salutejs/plasma-b2c@1.216.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-core@1.121.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-hope@1.216.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-icons@1.150.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-temple@1.165.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-tokens-b2b@1.21.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-tokens-b2c@0.30.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-tokens-web@1.36.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-tokens@1.55.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-ui@1.197.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-web@1.216.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-cy-utils@0.62.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-sb-utils@0.119.1-canary.543.5197189327.0
  yarn add @salutejs/plasma-tokens-utils@0.26.1-canary.543.5197189327.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-543/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-543/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-543/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-543/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-543/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-543/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: .github/workflows/examples-pr.yml [hunk 1]
```diff
@@ -1,95 +0,0 @@
...
-
-      - name: Lint
-        run: npm run lint
-
-      - name: Prepare directory for build
-        run: mkdir s3_build
-
-      - name: Build Demo Canvas App
-        run: |
-          npm run build --prefix="./examples/demo-canvas-app"
```
Comment (Yeti-or) [inline .github/workflows/examples-pr.yml:60]: тогда стоит сам пакет тоже дропнуть, все равно он не собирается

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-543/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-543/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-543/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-543/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-543/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-543/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by neretin-trike** [comment]
посмотри плс файл `clean-pr-documentation`, кажется, что в нём тоже идёт взаимодействие с example

**Comment by neretin-trike** [comment]
> @neretin-trike а что такое demo-tokens ?

хороший вопрос, с учётом, что там твои коммиты :D Ну вообще судя по ридми это просто какой-то пакет, на который мы ссылаемся в документации, чтобы показать, как работать с токенами. И тут я хз, надо ли оно там сейчас в таком виде

**Comment by Yeti-or** [comment]
@Yakutoc давай всю папку examples грохнем

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-543/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-543/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-543/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-543/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-543/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-543/temple-storybook/

**Review by kayman233 — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**